app/models/concerns/downloadable_file.rb

```diff
@@ -0,0 +1,28 @@
+module DownloadableFile
+  extend ActiveSupport::Concern
+
+  included do
+    has_attached_file :file, Settings.paperclip.to_hash
+
+    # TODO Limit attachment types for safe uploads
+    do_not_validate_attachment_file_type :file
+  end
+
+  def download_url
+    if fog?
+      file.send(:directory).files.get_url(
```

Comment:
Can we add some comments about what this is doing, and where this solution came from? Just because it feels a little weird.